doc/conf.py

```diff
@@ -149,6 +149,8 @@
     'dipy': ('https://dipy.org/documentation/1.2.0.', None),
     'mne_realtime': ('https://mne.tools/mne-realtime', None),
     'picard': ('https://pierreablin.github.io/picard/', None),
+    'qdarkstyle': ('https://github.com/ColinDuquesnoy/QDarkStyleSheet', None),
```

Comment:
I think these need to point to and `objects.inv` created by Sphinx, so I think this URL needs to exist:

But this one does:

https://qdarkstylesheet.readthedocs.io/en/latest/objects.inv

```suggestion
    'qdarkstyle': ('https://qdarkstylesheet.readthedocs.io/en/latest', None),
```
I don't think `darkdetect` has a Sphinx-built site so I don't think intersphinx will work, I would just use an anonymous RST URL for that...

Reply:
Thank you